src/Hl7.Fhir.Support.Poco/Introspection/FhirTypeAttribute.cs

```diff
@@ -68,6 +70,8 @@ public FhirTypeAttribute(string name, string canonical)
         /// <summary>
         /// The canonical of the StructureDefinition defining this type.
         /// </summary>
-        public string Canonical { get; set; }
+        public string? Canonical { get; set; }
```

Comment:
A nullable string type? that feels weird?

Reply:
This just means that the string MAY BE null, so the compiler can help you catch null reference exceptions: https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references

Reply:
The biggest null-reference ass-saver available (since C# 8.0)

Reply:
We are slowly annotating the whole SDK with these nullable annotations.  Microsoft has done the same for all their APIs starting from .NET 5.  If you see a file starting with `#nullable enable`, then you know we've edited it to use nullable annotations.